loops_labs.js

```diff
@@ -0,0 +1,103 @@
+//Write a while loop and a for loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+for (let num = 7; num > 0; num--) {
+  console.log(num);
+}
+
+console.log('-------------------');
+let numw = 7;
+while (numw <= 7 && numw > 0) {
```

Comment:
We can shorten this if statement into just this => if(numw >= 0). I would print out the same thing.